tests/test-cases/Group11-Upgrade/11-01-Upgrade.robot

```diff
@@ -63,6 +63,39 @@ Upgrade VCH with containers
     Should Not Contain  ${output}  Rolling back upgrade
     Should Be Equal As Integers  ${rc}  0
 
+    # check version
```

Comment:
So we test that the initial upgrade can be rolled back, and then that we can upgrade back to the new version from the rolled-back version. When I test ad-hoc I usually do this back and forth three or four times.